pkg/registry/csv.go

```diff
@@ -80,14 +80,15 @@ func ReadCSVFromBundleDirectory(bundleDir string) (*ClusterServiceVersion, error
 		}
 	}
 
+	csv := ClusterServiceVersion{}
```

Comment:
In line 69 above, it says `we will return the first file that is considered a CSV type`. That's no longer the behavior of this func as you will return error if there are multiple CSVs. Can you update the func comment?